packages/account-v2/src/modules/AddressFields/AddressFields.tsx

```diff
@@ -0,0 +1,70 @@
+import React from 'react';
+import { useAuthorize, useSettings, useStatesList } from '@deriv/api';
+import FormDropDownField from '../../components/FormFields/FormDropDownField';
+import FormInputField from '../../components/FormFields/FormInputField';
+import { addressDetailValidations } from './validations';
+
+export const AddressFields = () => {
+    const { data: activeAccount } = useAuthorize();
+    const { data: settings } = useSettings();
+
+    const { landing_company_name: landingCompanyName, upgradeable_landing_companies: upgradableLandingCompanies } =
+        activeAccount;
+
+    const isSvg = landingCompanyName === 'svg' || !!upgradableLandingCompanies?.includes('svg');
+    const { data: statesList, isFetched: statesListFetched } = useStatesList(settings.country_code || '', {
+        enabled: !!settings.country_code,
+    });
+
+    const {
+        addressCity: addressCitySchema,
+        addressLine1: addressLine1Schema,
+        addressLine2: addressLine2Schema,
+        addressPostcode: addressPostcodeSchema,
+        addressState: addressStateSchema,
+    } = addressDetailValidations(settings.country_code || '', isSvg);
+
+    return (
```

Comment:
just a suggestion:
maybe it would be nice to have some config array and just .map() it rather than add each FormInputField separately.
you can ignore this message :)

Reply:
IMO: It's better to keep the fields separately because it makes the code look cleaner and less conditions, however if we do have much more fields anywhere else it would make sense to add a config and map it.